File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithPresentation/ConvertingToPdf.php
<?php

namespace Aspose\Slides\WorkingWithPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class ConvertingToPdf {

    public static function run($dataDir=null)
    {
        # Instantiate a Presentation object that represents a PPTX file
        $pres = new Presentation($dataDir . "Aspose.pptx");

        # Saving the PPTX presentation to Pdf format
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Aspose.pdf", $save_format->Pdf);

        print "Document has been converted, please check the output file.";
    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithPresentation/ConvertingToPdf.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat

class ConvertingToPdf:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWithPresentation/ConvertingToPdf/'
                
        # Instantiate a Presentation object that represents a PPTX file
        pres = Presentation(dataDir + "Aspose.pptx")

        # Saving the PPTX presentation to Pdf format
        save_format = SaveFormat
        pres.save(dataDir + "Aspose.pdf", save_format.Pdf)

        print "Document has been converted, please check the output file."


if __name__ == '__main__':        
    ConvertingToPdf()